InferenceSystem/src/LiveInferenceOrchestrator.py

```
@@ -202,7 +211,21 @@ def get_config_path():
 			break
 		iteration_count += 1
 
-		clip_path, start_timestamp, current_clip_end_time = hls_stream.get_next_clip(current_clip_end_time)
+		try:
+			clip_path, start_timestamp, next_clip_end_time = hls_stream.get_next_clip(current_clip_end_time)
+		except (IndexError, ValueError) as e:
+			# Handle case when no audio files exist for the specified time range
+			print(f"\nWarning: Unable to retrieve audio clip. This may occur when no audio files exist for the specified time range.")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)